src/App.tsx

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    wasKeyPressed: false,
```

Comment:
Can you think what is the value of `keyPressed` when `wasKeyPressed === false`? That should point you into thinking, if you need this additional state property